src/_data/companies.yml

```diff
@@ -105,11 +105,40 @@ Pinterest:
   url: https://pinterest.com
   blog: https://medium.com/@Pinterest_Engineering
   description: "Pinterest is where people discover new ideas and find inspiration to do the things they love!"
-  
+
 Khan Academy:
   industry: Education
   url: https://www.khanacademy.org
   blog: http://engineering.khanacademy.org
   github: https://github.com/Khan
   location: Mountain View, California
   description: "Khan Academy offers a free, world-class education available. Serves some user-facing requests using a Kotlin-based application"
+
+Skookum:
+  industry: Consulting
+  url: https://skookum.com
+  github: https://github.com/Skookum
+  blog: https://skookum.com/blog
+  location: Denver, CO
+  description: "Skookum has the Product, Design, and Engineering know-how to solve your technical challenges"
+
+CyberCoders:
+  industry: Recruitment
+  url: https://www.cybercoders.com/
+  blog: https://www.cybercoders.com/blog/
+  location: Santa Clara, CA
+  description: "CyberCoders is a leading permanent placement recruiting firm. Through our innovative technology and efficient recruiters, CyberCoders helps find the right candidate fast."
+  jobs:
```

Comment:
@burden is this the correct job format?

Reply:
Format is correct. Rebase from master and you're good to merge 👍

Reply:
Nice. I wasn't sure how to do this.